### PR TITLE
Scala 2.13

### PR DESCRIPTION
**Pull request by phearnot**

### CODE REVIEW

File: lang/shared/src/main/scala/com/wavesplatform/lang/v1/repl/node/http/response/model/Transaction.scala [hunk 1]
```diff
@@ -92,7 +92,7 @@ object Transaction {
       version.toByte,
       senderPublicKey,
       proofs,
-      bodyBytes,
+      ByteString(),
```
Comment (msmolyakov) [inline lang/shared/src/main/scala/com/wavesplatform/lang/v1/repl/node/http/response/model/Transaction.scala:95]: how to get real bodyBytes?

Comment (phearnot) [reply lang/shared/src/main/scala/com/wavesplatform/lang/v1/repl/node/http/response/model/Transaction.scala:95]: https://jira.wavesplatform.com/browse/SC-631

File: lang/tests/src/test/scala/com/wavesplatform/lang/DirectiveParserTest.scala [hunk 2]
```diff
@@ -68,8 +68,8 @@ class DirectiveParserTest extends PropSpec with PropertyChecks with Matchers {
 
   property("directive with illegal format") {
     parse("{-# ILLEGAL #-}") shouldBe Left("Directive {-# ILLEGAL #-} has illegal format")
-    parse("{-# A%@#%$^$ A%#%%$ #-}") shouldBe 'left
-    parse("{-# AAA BBB CCC DDD #-}") shouldBe 'left
+    parse("{-# A%@#%$^$ A%#%%$ #-}") shouldBe Symbol("left")
```
Comment (msmolyakov) [inline lang/tests/src/test/scala/com/wavesplatform/lang/DirectiveParserTest.scala:71]: why can't we use `'left` now?

Comment (phearnot) [reply lang/tests/src/test/scala/com/wavesplatform/lang/DirectiveParserTest.scala:71]: Symbol literals are deprecated in 2.13

**Pull request merged**